cmd/cluster/logs/cmd.go

```diff
@@ -0,0 +1,145 @@
+package logs
+
+import (
+	"encoding/base64"
+	"fmt"
+	"github.com/spf13/cobra"
+	"os"
+)
+
+const (
+	DynatraceTenantKeyLabel    string = "sre-capabilities.dtp.tenant"
+	HypershiftClusterTypeLabel string = "ext-hypershift.openshift.io/cluster-type"
+	authURL                    string = "https://sso.dynatrace.com/sso/oauth2/token"
+	vaultPath                  string = "osd-sre/dynatrace/sd-sre-platform-oauth-client-grail"
+	vaultAddr                  string = "https://vault.devshift.net"
+)
+
+var (
+	dryRun        bool
+	hcp           bool
+	tail          int
+	hours         int
+	contains      string
+	cluster       string
+	sortOrder     string
+	namespaceList []string
+	nodeList      []string
+	podList       []string
+	containerList []string
+	statusList    []string
+)
+
+func NewCmdLogs() *cobra.Command {
+	logsCmd := &cobra.Command{
+		Use:   "logs",
+		Short: "Fetch logs from Dynatrace",
+		Args:  cobra.NoArgs,
+		RunE:  Run,
+	}
+
+	logsCmd.Flags().BoolVar(&dryRun, "dry-run", false, "Only builds the query without fetching any logs from the tenant")
+	logsCmd.Flags().IntVar(&tail, "tail", 10, "Last 'n' logs to fetch")
+	logsCmd.Flags().IntVar(&hours, "hours", 1, "Number of hours back to search")
```

Comment:
**nitpick:** would renaming this to `since` make more since? Then it would match other cmd line tools we use.